src/NuGet.Config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
is it needed for the currentversion?

Reply:
This is a mistake :/  I did not create this file myself. Do I need to remove this file from the PR or is it possible to merge without this?

Reply:
This is in the master branch. I don't understand why merging with master brought it into the diff.

Reply:
Opened #593.